### PR TITLE
feat(credentials): improve "Create Credentials Profile"

### PR DESCRIPTION
**Pull request by justinmk3**
## Problem:
The template for a new ~/.aws/credentials file is too verbose, which discourages reading of the most important parts, and makes it harder to see which fields need to be edited if starting with a "blank" template.

## Solution:
Remove non-essential discursions. Reword for clarity and brevity.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.quickstart.vscode.md [hunk 1]
```diff
@@ -247,6 +247,7 @@ palette_, available by selecting _View > Command Palette_ or by typing
...
 | `AWS: Deploy SAM Application`                        | Deploys a local serverless application to an AWS account. For more information, see [Deploying a Serverless Application](https://docs.aws.amazon.com/console/toolkit-for-vscode/deploy-serverless-app) in the user guide.                                                                                      |
 | `AWS: Detect SAM CLI`                                | Checks whether the Toolkit can communicate correctly with the AWS SAM CLI that is installed.                                                                                                                                                                                                                   |
+| `AWS: Edit Credentials`                              | Opens the `~/.aws/credentials` or `~/.aws/config` file for editing.                                                                                                                                                                                                                                            |
```
Comment (JadenSimon) [inline README.quickstart.vscode.md:250]: Potential idea: leverage `Commands` to generate this file ?
We could make each command name become a clickable link.

**Review by JadenSimon — APPROVED** [review]

**Comment by JadenSimon** [comment]
FYI the tests are failing because they are ran with 0 profiles. Which means the failures are actually a good thing because that means the feature is working correctly?

To fix, just add a dummy profile for the failing tests:
```ts
        const wizard = new CreateProfileWizard({ foo: {} }, template)
```

And maybe add a new test to cover the new behavior.

File: README.quickstart.vscode.md (hunk 1 above)
Comment (justinmk3) [reply README.quickstart.vscode.md:250]: > Potential idea: leverage Commands to generate this file ?

Definitely! Forgot that is another reason we needed the commads abstraction :)

File: src/test/credentials/wizards/createProfile.test.ts [hunk 2]
```diff
@@ -34,20 +34,26 @@ class TestPrompter extends Prompter<string | undefined> {
...
+        const tester = createWizardTester(new CreateProfileWizard({ foo: {} }, processCredentialsTemplate))
         tester.name.assertShowFirst()
         tester.profile[ProfileKey.Process].assertShowSecond()
         tester.accountId.assertShowThird()
     })
 
+    it('skips profile name step (and uses "default") if starting with no profiles', function () {
+        const tester = createWizardTester(new CreateProfileWizard({}, processCredentialsTemplate))
+        tester.name.assertDoesNotShow()
+        tester.assertShowCount(2)
```
Comment (JadenSimon) [inline src/test/credentials/wizards/createProfile.test.ts:54]:
```suggestion
        tester.name.assertDoesNotShow()
        tester.name.assertValue('default')
        tester.assertShowCount(2)
```

**Review by JadenSimon — APPROVED** [review]

**Pull request merged**